samples/cmaf/ham/cmaf-ham-conversion/package.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "cmaf-ham-conversion",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "dev": "node --loader ts-node/esm src/index.ts",
+        "serve": "serve --cors ./output"
+    },
+    "dependencies": {
+        "@svta/common-media-library": "file:../../../../../../lib/",
```

In a monorepo, this should be be:
```suggestion
        "@svta/common-media-library": "*",
```